### ISSUE TITLE
minify all enqueued scripts and styles

### ISSUE THREAD

**Issue by falcoraptor** [ThematicTheme/Thematic#58]
Keep original for reference and enqueue the minified.

```
file.css       <---- with plenty of comments
file.min.css   <------ enqueued file
```

**Comment by scottnix** [comment]
I would almost say to push this issue way down on the list of features to implement.

For anyone using Sass, this isn't useful. An extra few steps would most likely need to be added to Sass to get it to save the file name with the .min. portion of the file extension.

Also I hope most people use a WordPress Plugin like W3 Total Cache which does minify, so it may be unnecessary to worry about this too much.

**Comment by JamesGeiger** [comment]
Still a good idea though to minify it.

**Comment by falcoraptor** [comment]
Yes minifying will definitely be something done towards the end of the release cycle. But I like it because

a) It might discourage tampering with the parent css file and promote proper use of a child theme and b) I want to make the parent theme as small and fast as possible. 

I agree that the best option for many sites is a plugin like W3 total cache, but it doesn't hurt to have the parent theme defaults with best performance in mind.
